lib/platform/gitea/gitea-got-wrapper.spec.ts

```diff
@@ -1,7 +1,7 @@
-import { GotResponse } from '../../../lib/platform';
-import { partial } from '../../util';
-import { GotFn } from '../../../lib/util/got';
-import { GiteaGotApi } from '../../../lib/platform/gitea/gitea-got-wrapper';
+import { GotResponse } from '..';
+import { partial } from '../../../test/util';
```

Comment:
Need to work out what to do about this util file

Reply:
yes please suggest, until then keeping as is.